OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/PreferenceCenterConfig.java

```diff
@@ -0,0 +1,41 @@
+package com.optimove.android;
+
+import androidx.annotation.Nullable;
+
+public class PreferenceCenterConfig {
+    @Nullable
+    private String region;
+
+    @Nullable
+    private String brandGroupId;
+
+    @Nullable
+    private int tenantId;
+
+    public PreferenceCenterConfig() {}
+
+    @Nullable
+    public String getRegion() {
```

Comment:
all these ones nullable and have public setters. But region/tenant id/brandGroupId cannot change. (Later possible that we allow updating config after initialisation, but again, we just update entire config creating a new one)

---> we could make them final, nonnull and assigned in the constructor?